src/ast/nodes/CallExpression.ts

```diff
@@ -248,7 +248,14 @@ export default class CallExpression extends NodeBase implements DeoptimizableEnt
 					for (let index = 0; index <= lastIncludedIndex; index++) {
 						this.arguments[index].render(code, options);
 					}
-					code.remove(this.arguments[lastIncludedIndex].end, this.end - 1);
+					code.remove(
+						findFirstOccurrenceOutsideComment(
+							code.original,
+							',',
+							this.arguments[lastIncludedIndex].end
+						),
+						this.end - 1
+					);
```

Comment:
Nice one, I'll take yours 😜

Reply:
It was a simple enough issue to debug! as usual https://astexplorer.net/ was my friend

Reply:
It is mine as well 😉